### PR TITLE
3D Tiles Cleanup

### PR DESCRIPTION
**Pull request by lilleyse**
Based on review comments in #5308 

I think I got mostly everything here. Will leave more notes on the other PR.

### CODE REVIEW

File: Specs/Scene/Instanced3DModel3DTileContentSpec.js [hunk 1]
```diff
@@ -67,6 +67,10 @@ defineSuite([
         scene.primitives.removeAll();
     });
 
+    it ('gets properties', function() {
```
Comment (pjcozzi) [inline Specs/Scene/Instanced3DModel3DTileContentSpec.js]: What's the plan here?

**Comment by pjcozzi** [comment]
LGTM

Let's aim to merge this today.

@mramato do you want to look at this?

File: Specs/Scene/Instanced3DModel3DTileContentSpec.js (hunk 1 above)
Comment (lilleyse) [inline Specs/Scene/Instanced3DModel3DTileContentSpec.js]: Oh, forgot to remove that. Updated now.

File: Source/Scene/Cesium3DTileStyle.js [hunk 2]
```diff
@@ -67,39 +78,30 @@ define([
...
+            promise = RequestScheduler.request(style, loadJson);
         } else {
-            setup(style, data);
-            style._readyPromise.resolve(style);
+            promise = when.resolve(style);
         }
+
+        promise.then(function(styleJson) {
+            setup(that, styleJson);
+            that._readyPromise.resolve(that);
```
Comment (mramato) [inline Source/Scene/Cesium3DTileStyle.js]: This is redundant, you can just assign `this._readyPromise = promise.then....` and avoid using the defer completely. I'll just submit a tweak myself so we can get this merged.

**Comment by mramato** [comment]
I'll admit, I didn't review this rigorously, but it looks good to me.

There is a release only test failure for `tile expires` (not sure if it's sporadic or happens every time).  I'm okay with merging this and fixing the release failure anytime before 3d-tiles goes into master.

@pjcozzi feel free to merge.

**Comment by mramato** [comment]
And thanks @lilleyse!

**Pull request merged**